### DISCUSSION TITLE
Allow removal of failed tweak downloads/install from the install queue without stopping the rest similar to how Sileo works.

### DISCUSSION THREAD

**Discussion by Garydunlop00** [wstyres/Zebra#1443] [Ideas]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by absidue** [comment]
You can swipe right to left on any package in the queue to remove it from the queue.

**Comment by Garydunlop00** [comment]
> You can swipe right to left on any package in the queue to remove it from the queue.

That’s not what is asked for....

In Sileo if a tweak fails to download/ Install you can remove it from the queue while the rest continue to download.

In Zebra if one tweak fails to download/ Install then you have to stop the full queue remove the tweak that failed and then try again.
